src/pages/OrderDetails.js

```diff
@@ -12,29 +13,34 @@ export default function OrderDetails({
         <h3>Prüfe Deine Daten und bestätige bitte den Kauf:</h3>
         <main>
           {" "}
-          <div>
+          <div className="allInputs">
             <h4>
-              Name: <span>{inputData.name} </span>
+              <p className="inputs">Name: </p>{" "}
```

Comment:
Wieso nicht styled components?

Reply:
Why not? It is styled components combined with normal css. All main components:  Section, Article, NextButtons are styled comp. and some(not all) little tags inside are written with normal css.

Reply:
Ist das für die Coaches okay? Von meiner Seite her kein Problem.

Reply:
Not the best solution to mix classes with styled components. Your `section` in this file is a rather large chunk. It's so large because you decided to style every nested element within that section. A better approach would be to take smaller chunks. Take a section element, for instance, and style it. Take a paragraph and style it. You'll have more styled components that way, yes, but it's also less cluttered and, more importantly, it works in isolation.

Reply:
hi, ok I will change it!

Reply:
@Kheirah Hi Ernst, i  have already changed it!

Reply:
I can not write <Inputs DataInputs Address>...</Inputs DataInputs Address>

Reply:
You write a styled component for the `p` tag and write down everything that you wrote for the classes "inputs dataInputs address".